python/packages/ai/teams/ai/clients/llm_client.py

```diff
@@ -284,7 +291,20 @@ def chunk_received(
 
             if streamer is not None:
                 await streamer.end_stream()
```

Comment:
also need to remove this line

Reply:
does this not align with this line from Steve's updates?
https://github.com/microsoft/teams-ai/blob/0b10ad82f992e959bd2b5729e19a7422785ccc78/js/packages/teams-ai/src/planners/LLMClient.ts#L378

else {
                    if (response.status == 'success') {
                        // Delete message from response to avoid sending it twice
                        delete response.message;
                    }

                    // End the stream and remove pointer from memory
                    // - We're not listening for the response received event because we can't await the completion of events.
                    await streamer.endStream();
                    memory.deleteValue('temp.streamer');
                }

Reply:
oh I may have the wrong place